src/node/context.h

```diff
@@ -16,13 +16,14 @@ class CAddrMan;
 class CBlockPolicyEstimator;
 class CConnman;
 class CCreditPoolManager;
-struct CJContext;
+class ChainstateManager;
+class CEvoDB;
 class CScheduler;
 class CTxMemPool;
-class ChainstateManager;
-struct LLMQContext;
 class PeerManager;
-class CEvoDB;
+struct CJContext;
```

Comment:
I think better to order not pure alphabetically like `class < struct` but by class/struct name: `class A < struct B` and `struct C < class D`

Reply:
Resolved in latest push